examples/raspberrypi/rp2xxx/src/interrupts.zig

```diff
@@ -59,22 +70,9 @@ pub fn main() !void {
 
     set_alarm(1_000_000);
 
-    switch (rp2xxx.compatibility.chip) {
-        .RP2040 => {
-            peripherals.TIMER.INTE.toggle(.{ .ALARM_0 = 1 });
-            interrupt.enable(.TIMER_IRQ_0);
-        },
-        .RP2350 => {
-            peripherals.TIMER0.INTE.toggle(.{ .ALARM_0 = 1 });
-            switch (rp2xxx.compatibility.arch) {
-                .arm => interrupt.enable(.TIMER0_IRQ_0),
-                .riscv => {
-                    interrupt.enable(.MachineExternal);
-                    interrupt.external.enable(.TIMER0_IRQ_0);
-                },
-            }
-        },
-    }
+    timer.INTE.toggle(.{ .ALARM_0 = 1 });
+
+    interrupt.enable(timer_irq);
```

Comment:
riscv doesn't need `interrupt.external.enable`?

Reply:
It does internally, but it is done as part of system setup in my hazard3.zig by the `_start_c` function.  

The goal was to make handing NVIC interrupts (the thing most likely needed in user code) identical between the riscv and arm cores.